packages/env/README.md

```diff
@@ -175,6 +175,30 @@ Positionals:
             [string] [choices: "all", "development", "tests"] [default: "tests"]
 ```
 
+### `wp-env run [container] [command]` 
+
+```sh
+wp-env run <container> [command..]
+
+Runs an arbitrary command in one of the underlying Docker containers.
```

Comment:
It would be nice for those Ctrl+F ing through docs to mention here "for example running wp cli commands". I know you added this to the exanples, it would be nice make the connection immediately.